reframe/frontend/executors/policies.py

```diff
@@ -416,9 +464,10 @@ def exit(self):
         pollrate = PollRateFunction(0.2, 60)
```

Comment:
To do this I would need to run many long testcases, right? I can get the current polling rate from jenkins, from the reframe.log. Wouldn't it be better if we leave it as it is now and see how many polls are performed after the pr is merged? Because for a few short jobs it wouldn't make a difference, right?

Reply:
Now, it's much easier to estimate the poll rate, because it does not depend any more on the number of running tasks. If we keep the poll rate function as of now, we will need to refactor it, because I've seen that you just copied it in two different places.